.github/changes-filter.yaml

```diff
@@ -10,3 +10,5 @@ docs:
   - '**/*.md'
 tests:
   - added|modified: '**/src/test/java/**/*.java'
+cpp:
+  - 'pulsar-client-cpp/**'
```

Comment:
We should also run the cpp test while the changes are introduced to the broker and proxy.

Reply:
Yes, we will run them. This PR will not affect this part. It will only affect if the changes are all in the `pulsar-client-cpp/**`